lib/hydra/works/models/concerns/block_child_objects.rb

```diff
@@ -0,0 +1,21 @@
+module Hydra::Works
+  # Do not allow aggregation of child objects
+  module BlockChildObjects
+
+    def child_objects= objects
+      raise NoMethodError, "undefined method `child_objects=' for #{self}"
```

Comment:
Yep, it is a sort of lie.  Do you have a suggestion of an alternate error to raise?